components/script/dom/xmlhttprequest.rs

```diff
@@ -979,14 +979,12 @@ impl XMLHttpRequest {
                 // Part of step 11, send() (processing response end of file)
                 // XXXManishearth handle errors, if any (substep 2)
 
-                // Subsubsteps 5-7
+                // Subsubsteps 6-8
                 self.send_flag.set(false);
 
                 self.change_ready_state(XMLHttpRequestState::Done);
                 return_if_fetch_was_terminated!();
-                // Subsubsteps 10-12
-                self.dispatch_response_progress_event(atom!("progress"));
```

Comment:
Looking a bit more into details, I think this change is conforming to the spec. The ```XHRProgress::Done``` is only passed when we detect that the transmittion is done (so a progress event has been emitted with the total bits transmitted). According to this [note](https://github.com/whatwg/xhr/blob/master/xhr.bs#L938) and this isssue https://github.com/whatwg/xhr/issues/72, we can't emit a new process event if we don't have new bits which we don't have because the transmission is already over.